projects/igniteui-angular/src/lib/grids/grid/grid.component.html

```diff
@@ -100,7 +100,7 @@
         | gridPostGroupBy:groupingExpressions:groupingExpansionState:groupsExpanded:id:pipeTrigger
         | gridSummary:hasSummarizedColumns:summaryCalculationMode:summaryPosition:id:pipeTrigger:summaryPipeTrigger"
         let-rowIndex="index" [igxForScrollOrientation]="'vertical'"  [igxForScrollContainer]='verticalScroll'
-        [igxForContainerSize]='calcHeight' [igxForItemSize]="hasColumnLayouts ? rowHeight * multiRowLayoutRowSize : rowHeight" #verticalScrollContainer (onChunkPreload)="dataLoading($event)">
+        [igxForContainerSize]='calcHeight' [igxForItemSize]="hasColumnLayouts ? renderedRowHeight * multiRowLayoutRowSize : renderedRowHeight" #verticalScrollContainer (onChunkPreload)="dataLoading($event)">
```

Comment:
This is not correct for the MRL grid. For it the 1px additional size is set for the whole row, not per row spanned cell. So here the size passed for igxForItemSize should be  - rowHeight *multiRowLayoutRowSize  + 1.